x-pack/plugins/ml/public/shared_components/single_metric_viewer/single_metric_viewer.tsx

```diff
@@ -107,35 +113,17 @@ const SingleMetricViewerWrapper: FC<SingleMetricViewerPropsWithDeps> = ({
 
   const previousRefresh = usePrevious(lastRefresh ?? 0);
 
-  useEffect(
-    function setUpJobsLoaded() {
-      async function loadJobs() {
-        try {
-          await mlJobService.loadJobsWrapper();
-          setJobsLoaded(true);
-        } catch (e) {
-          if (onError) {
-            onError(new Error(errorMessage));
-          }
-        }
-      }
-      if (isMounted() === false) {
-        return;
-      }
-      loadJobs();
-    },
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-    [isMounted]
-  );
-
   useEffect(
     function setUpSelectedJob() {
       async function fetchSelectedJob() {
         if (mlApiServices && selectedJobId !== undefined) {
           try {
-            const { jobs } = await mlApiServices.getJobs({ jobId: selectedJobId });
+            const [{ jobs }, { jobs: jobStats }] = await Promise.all([
+              mlApiServices.getJobs({ jobId: selectedJobId }),
+              mlApiServices.getJobStats({ jobId: selectedJobId }),
+            ]);
             const job = jobs[0];
-            setSelectedJob(job);
+            setSelectedJobWrapper({ job, stats: jobStats[0] });
```

Comment:
Super nit, for consistency :) 
```suggestion
            setSelectedJobWrapper({ job: jobs[0], stats: jobStats[0] });
```

Reply:
Updated in 53b573ff6d8d57acfe1ca468df74f5a8d63354a3